string drone_sn
uint32 cargo_id